NETWORK_MASS_SPAWN_MITIGATIONS.md

```
@@ -0,0 +1,54 @@
+## Server-side throttles (implemented)
+
+| Property | Default | Description | Live command |
+|----------|---------|-------------|--------------|
+| `monster_tick_throttle_limit` | 75 | Monsters processed per tick per landblock | `/modifylong monster_tick_throttle_limit 50` |
+| `action_queue_throttle_limit` | 300 | Actions processed per tick | `/modifylong action_queue_throttle_limit 150` |
+
+---
+
+## Suggested values for mass spawns (500+ mobs)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)